### PR TITLE
Filter My Forms list to not show forms you can just launch

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1768

Don't throw all the forms that you can only launch into the forms list. Those can be accessed by the share link.
We may revisit this in the future for a forms catalogue or a User forms screen.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-105

File: app/frontend/tests/unit/store/modules/form.getters.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    store = new Vuex.Store(cloneDeep(formStore));
+    store.replaceState(cloneDeep(sampleState));
+  });
+
+  it('form should return the state form', () => {
+    expect(store.getters.form).toEqual(sampleState.form);
+  });
+
+  it('formlist should return the state formlist', () => {
+    expect(store.getters.formlist).toEqual(sampleState.formlist);
```
Comment (TimCsaky) [inline app/frontend/tests/unit/store/modules/form.getters.spec.js:45]: sample.form**L**ist

Comment (loneil) [reply app/frontend/tests/unit/store/modules/form.getters.spec.js:45]: Ohhh **that's** why coverage isn't showing on that getter. Thx!

**Review by TimCsaky — APPROVED** [review]
nice!

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-105

**Review by jujaga — APPROVED** [review]

**Pull request merged**